src/components/DropList/DropList.Combobox.jsx

```diff
@@ -32,6 +33,7 @@ function Combobox({
   'data-cy': dataCy = `DropList.${VARIANTS.COMBOBOX}`,
   focusToggler = noop,
   handleSelectedItemChange = noop,
+  hideCustomListIfEmptyInput = false,
```

Comment:
I set a default value at the top level of `DropList`; should I not set it here?

Reply:
that's fine, all props here has a default value, let's keep the same pattern